### PR TITLE
Fbs wayf registration

### PR DESCRIPTION
**Pull request by xendk**
Create user support for FBS.

Also minor cleanups (an exception class that was not used, for instance).

### CODE REVIEW

File: modules/ding_provider/ding_provider.exceptions.inc [hunk 1]
```diff
@@ -80,13 +80,3 @@ class DingProviderUserException extends Exception {
     }
   }
 }
-
```
Comment (cableman) [inline modules/ding_provider/ding_provider.exceptions.inc]: It this exception no longer used by the providers ?

File: modules/fbs/prancer/SwaggerApiError.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Reload\Prancer;
```
Comment (cableman) [inline modules/fbs/prancer/SwaggerApiError.php]: This is for sure outside this PR's scope, but should the namespace have been DDB and not Reload?

**Comment by cableman** [comment]
Reviewed with minor comment.

File: modules/ding_provider/ding_provider.exceptions.inc (hunk 1 above)
Comment (xendk) [inline modules/ding_provider/ding_provider.exceptions.inc]: Never have been. It provides no functionality, it has a private property nobody uses, and the only one using it was ding_user catching it.

But ding_user should catch any exception, and as nobody is throwing it anyway, I removed it.

File: modules/fbs/prancer/SwaggerApiError.php (hunk 2 above)
Comment (xendk) [inline modules/fbs/prancer/SwaggerApiError.php]: Do DDB want to maintain a generic Swagger spec to PHP code generator?

**Comment by xendk** [comment]
Rebased.

File: modules/ding_provider/ding_provider.exceptions.inc (hunk 1 above)
Comment (cableman) [inline modules/ding_provider/ding_provider.exceptions.inc]: Okay, super :-)

File: modules/fbs/prancer/SwaggerApiError.php (hunk 2 above)
Comment (cableman) [inline modules/fbs/prancer/SwaggerApiError.php]: I have not used Swagger, so I don't know... so I think that would be a: No

**Comment by cableman** [comment]
Reviewed and awaiting next release

**Pull request merged**